sctp_linux.go

```diff
@@ -253,6 +254,20 @@ func listenSCTPExtConfig(network string, laddr *SCTPAddr, options InitMsg, contr
 		nil
 }
 
+// FileListener takes a file, dup's the underlying file descriptor, and returns
+// a SCTPListener created from the dup'd fd.
+func FileListener(file *os.File) (*SCTPListener, error) {
+	r1, _, err := syscall.Syscall(syscall.SYS_FCNTL, file.Fd(), syscall.F_DUPFD_CLOEXEC, 0)
```

Comment:
@akerouanton What do you think about using `unix.FcntlInt` instead of `syscall.Syscall`?

Reply:
@ishidawataru WDYT? Do you prefer adding `golang.org/x/sys` as a dep?

Reply:
I'm sorry for the delay. It makes sense to avoid adding a dependency. I'm merging this PR. Thank you for your contribution.